public/locales/fr/common.json

```
@@ -0,0 +1,56 @@
+{
+  "scrobble": "Scrobble",
+  "getScrobbling": "Début du scrobbling",
+  "loading": "Chargement...",
+  "artist": "Artiste",
+  "song": "Musique",
+  "title": "Titre",
+  "album": "Album",
+  "albumArtist": "Artiste de l'album",
+  "history": "Historique",
+  "yourProfile": "Votre profil",
+  "language": "Langage",
+  "translateYourLanguage": "Vous voulez ajouter votre langage?",
+  "logIn": "Se connecter",
+  "logOut": "Se déconnecter",
+  "now": "Maintenant",
+  "custom": "Personnaliser...",
+  "timestamp": "Timestamp",
+  "lastfmWillRejectOldTimestamps": "Last.fm rejecte les scrobbles datant de plus de deux semaines.",
+  "noSongsScrobbled": "Aucune musique n'a été scrobblé !",
+  "songsWillAppearHere": "Les pistes apparaîtront ici une fois que vous les aurez scrobblés.",
+  "clearHistory": "Nettoyer l'historique ",
+  "scrobbleAgain": "Scrobbler encore",
+  "copyToEditor": "Le copier à l'éditeur",
+  "swapArtistTitle": "Échanger l'artiste et le morceau",
+  "viewProfile": "Voir mon profil Last.fm",
+  "lockArtist": "Verrouiller le champ de l'artiste",
+  "lockAlbum": "Verrouiller le champ de l'album",
+  "pasteIntercepted": "Le texte collé a été divisé automatiquement !",
+  "considerDonating": "S'il vous plait supporter le projet sur Patreon !",
```

May we make this one in imperative form?

```suggestion
  "considerDonating": "S'il vous plait supportez le projet sur Patreon !",
```

---

Yes 👍